pkg/agent/nodeportlocal/portcache/port_table_windows_test.go

```diff
@@ -0,0 +1,94 @@
+//go:build !windows
+// +build !windows
+
+// Copyright 2021 Antrea Authors
```

Comment:
s/2021/2022/

Reply:
thanks for reminding will make these changes when making PR ready for review.